iot-hub/Samples/service/DigitalTwinClientSamples/Thermostat/ThermostatSample.cs

```diff
@@ -0,0 +1,136 @@
+
+            _logger.LogDebug($"Update {_digitalTwinId} digital twin response: {updateDigitalTwinResponse.Response.StatusCode}.");
+        }
+
+        private async Task UpdateCurrentTemperatureProperty()
+        {
+            // Choose a random value to assign to the currentTemperature property
+            int currentTemperature = Random.Next(0, 100);
+
+            const string currentTemperaturePropertyName = "currentTemperature";
+            var updateOperation = new UpdateOperationsUtility();
+
+            // First, add the property to the digital twin
+            updateOperation.AppendAddOp($"/{currentTemperaturePropertyName}", currentTemperature);
+            _logger.LogDebug($"Add the {currentTemperaturePropertyName} property on the {_digitalTwinId} digital twin with a value of {currentTemperature}.");
+            HttpOperationHeaderResponse<DigitalTwinUpdateHeaders> addPropertyToDigitalTwinResponse = await _digitalTwinClient.UpdateDigitalTwinAsync(_digitalTwinId, updateOperation.Serialize());
+            _logger.LogDebug($"Update {_digitalTwinId} digital twin response: {addPropertyToDigitalTwinResponse.Response.StatusCode}.");
+            // Print the Thermostat digital twin
+            await GetAndPrintDigitalTwin<ThermostatTwin>();
+
+            // Second, update the property to a different value
+            int newCurrentTemperature = Random.Next(0, 100);
+            updateOperation.AppendAddOp($"/{currentTemperaturePropertyName}", newCurrentTemperature);
+            _logger.LogDebug($"Replace the {currentTemperaturePropertyName} property on the {_digitalTwinId} digital twin with a value of {newCurrentTemperature}.");
+            HttpOperationHeaderResponse<DigitalTwinUpdateHeaders> replacePropertyInDigitalTwinResponse = await _digitalTwinClient.UpdateDigitalTwinAsync(_digitalTwinId, updateOperation.Serialize());
+            _logger.LogDebug($"Update {_digitalTwinId} digital twin response: {replacePropertyInDigitalTwinResponse.Response.StatusCode}.");
+            // Print the Thermostat digital twin
+            await GetAndPrintDigitalTwin<ThermostatTwin>();
+
+            // Third, remote the currentTemperature property
+            updateOperation.AppendRemoveOp($"/{currentTemperaturePropertyName}");
+            _logger.LogDebug($"Remove the {currentTemperaturePropertyName} property on the {_digitalTwinId} digital twin.");
+            HttpOperationHeaderResponse<DigitalTwinUpdateHeaders> removePropertyInDigitalTwinResponse = await _digitalTwinClient.UpdateDigitalTwinAsync(_digitalTwinId, updateOperation.Serialize());
+            _logger.LogDebug($"Update {_digitalTwinId} digital twin response: {removePropertyInDigitalTwinResponse.Response.StatusCode}.");
+            // Print the Thermostat digital twin
+            await GetAndPrintDigitalTwin<ThermostatTwin>();
+        }
```

Comment:
super nitty: what do you think about adding something that shows how to do multiple patch operations through a single Update call?
meaning, the updateOperations will have both a replace and an add (could be for different properties), but just to highlight that the `Update` API takes in a list of update operations => you can pass in more than one.

This might be a better idea for the temperature controller one -> we have multiple components with multiple properties available to us there.
Thoughts?

Reply:
Good idea. I will leave this comment open to get to it after all higher priority items are resolved.